packages/testkit/utils/types.ts

```diff
@@ -0,0 +1,18 @@
+export type BoundingBox = {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+};
+
+export type MouseButton = "left" | "right" | "middle";
+
+export type WheelScrollPixels = {
+  deltaX: number;
+  deltaY: number;
+};
+
+export type Coordinates = {
+  x: number;
+  y: number;
+};
```

Comment:
those also available in the native types, consider using them later
not a blocker for now